vertical-pod-autoscaler/pkg/updater/priority/update_priority_calculator.go

```diff
@@ -140,7 +140,7 @@ func (calc *UpdatePriorityCalculator) AddPod(pod *apiv1.Pod, now time.Time) {
 		klog.V(4).Infof("not updating pod %v/%v because resource would not change", pod.Namespace, pod.Name)
 		return
 	}
-	klog.V(2).Infof("pod accepted for update %v/%v with priority %v", pod.Namespace, pod.Name, updatePriority.ResourceDiff)
+	klog.V(2).Infof("pod accepted for update %v/%v with priority %v", pod.Namespace, pod.Name, updatePriority.ResourceDiff, processedRecommendation)
```

Comment:
I built the `vpa-updater` with your changes and looked at the logs during runtime:
```
I0419 08:53:06.733049       1 update_priority_calculator.go:143] pod accepted for update default/hamster-c6967774f-h4xj5 with priority 5.27%!(EXTRA *v1.RecommendedPodResources=&{[{hamster map[cpu:{{627 -3} {<nil>} 627m DecimalSI} memory:{{262144 3} {<nil>} 262144k DecimalSI}] map[
cpu:{{115 -3} {<nil>} 115m DecimalSI} memory:{{262144 3} {<nil>} 262144k DecimalSI}] map[cpu:{{1 0} {<nil>} 1 DecimalSI} memory:{{524288000 0} {<nil>} 500Mi BinarySI}] map[cpu:{{627 -3} {<nil>} 627m DecimalSI} memory:{{262144 3} {<nil>} 262144k DecimalSI}]}]})
```

I'm not sure the formatting makes it easy to see what's really going on. It is pretty messy to parse as a human and probably also not very easy to process for scripts. Even for this simple case with the `hamster` example, where we only have recommendations for a single container. For Pods with multiple containers, this will get out of hand quickly. We probably don't care about `lowerBound` and `upperBound` here, but most likely about `target` and `uncappedTarget` and potentially for all containers.

Reply:
Good point - I pushed a commit that implements your suggestion to have a separate func that formats the `RecommendedPodResources`'s `target` and `uncappedTarget` 👍